changelog.asc

```diff
@@ -2,6 +2,44 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241025)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Anki upstream version 24.06.3 included
```

Comment:
+ FSRS version

Reply:
I'll add only the version. The changelog has other  versions declared and someone going there might say that we are way behind.